### PR TITLE
Fix links to ECS module

### PR DESCRIPTION
**Pull request by lkysow**

### CODE REVIEW

File: website/content/docs/ecs/get-started/install.mdx [hunk 1]
```diff
@@ -243,7 +243,7 @@ module "my_task" {
 }
 ```
 
-All possible inputs are documented on the [module reference documentation](https://registry.terraform.io/modules/hashicorp/consul/aws-ecs/latest/submodules/mesh-tas?tab=inputs)
+All possible inputs are documented on the [module reference documentation](https://registry.terraform.io/modules/hashicorp/consul-ecs/aws/latest/submodules/mesh-tas?tab=inputs)
```
Comment (BeenzSyed) [inline website/content/docs/ecs/get-started/install.mdx:246]: I think there is a 'k' missing for 'mesh-tas' in this link.

I believe it should be 'https://registry.terraform.io/modules/hashicorp/consul-ecs/aws/latest/submodules/mesh-task?tab=inputs'

Comment (lkysow) [reply website/content/docs/ecs/get-started/install.mdx:246]: 🦅 👀

**Review by BeenzSyed — APPROVED** [review]
LGTM! Thank you Luke!

**Comment by hc-github-team-consul-core** [comment]
:cherries: If backport labels were added before merging, cherry-picking will start automatically.

To retroactively trigger a backport after merging, add backport labels and re-run https://circleci.com/gh/hashicorp/consul/389548.

**Comment by hc-github-team-consul-core** [comment]
:cherries::white_check_mark: Cherry pick of commit a7e7c15adfb3ab443949a803d00cdb6eb2841f66 onto `stable-website` succeeded!

**Comment by hc-github-team-consul-core** [comment]
:cherries::white_check_mark: Cherry pick of commit a7e7c15adfb3ab443949a803d00cdb6eb2841f66 onto `release/1.10.x` succeeded!

**Pull request merged**